### DISCUSSION TITLE
Access to USB port

### DISCUSSION THREAD

**Discussion by silentnelis** [tasmota/tasmotizer#67] [Q&A] [answered]
I’m a beginner and need some help, I try to flash Sonoff S20 device through Tasmotizer 1.2.1 on a mac (Big Sur) but one way ore the other the Tasmotizer doesn't see the usb port after refreshing. Driver installed and hardware USB device Tree even shows FT232 USB UART. 
A strange thing is that if i'm looking at Hardware at the USB Device tree I see that FT232 USB UART showing in the USB 3.0 Bus section while my Wacom intuos pro connection is in the USB2.0 Hub section.
Any ideas?

**Comment by silentnelis** [comment]
Additional pictures
<img width="487" alt="Screenshot 2020-12-18 at 12 19 27" src="https://user-images.githubusercontent.com/68472089/102609360-9c808680-412b-11eb-9b59-b5ff10b61eb7.png">

**Comment by Jason2866** [comment, marked as answer]
It is a issue from Big Sur. Update to latest development version of pyserial.
https://github.com/tasmota/tasmotizer/issues/65

**Comment by jziolkowski** [comment]
Post a reply if what Jason2866 suggested worked, then the issue will be closed as fixed.

**Reply by silentnelis** [reply]
I updated, actually re-installed Tasmotizer py version 3.5 but it still doesn't work. In the meantime I flashed my Sonoff device with a Raspberry Pi that worked well. Thanks for your replies anyway.

**Comment by Jason2866** [comment]
I move it to discussions.
